### ISSUE TITLE
P-values = 1 lead to infinite se column in *z file

### ISSUE THREAD

**Issue by sinarueeger** [FINNGEN/finemapping-pipeline#23]
I am fine-mapping meta-analysed GWASs (two GWASs from FinnGen). 

Some of the variants have a P-value of exactly 1. Because of reweighting the standard errors, this leads to infinite entries in the `se` column of the *z file, and ultimately `susie_suff_stat` errors out with the error message 
```
Error in susie_suff_stat(bhat = beta, shat = se, R = R, n = n, var_y = var_y,  :
  shat contains zero.
```

**Comment by Fedja** [comment]
Oh surprising division by zero creates inf instead of erroring out in numpy.
Fixed in here #24, pull request for @mkanai

**Comment by Fedja** [comment]
how have we never had even a single case of p=1?!?!?

**Comment by mkanai** [comment]
I think partly because we pre-filter to genome-wide significant loci...

**Comment by mkanai** [comment]
fixed in #24
